game/games/flappybird/game.cfg

```
@@ -0,0 +1,15 @@
+; This example file lists all required and optional Values
+; It is meant for easy identification of games. 
+
+[game]
+; Choose a memorable name
+name ="Flappy Bird"
```

```suggestion
name="Flappy Bird"
```
The repo will be haunted by this typo forever...

---

i will add some, thanks for the feedback!